doc/Migrate.md

```diff
@@ -0,0 +1,81 @@
+Migrating to a new server installation
+===============
+
+* [Home](help)
+
+## Preparation
+
+### New server
+Set up your new server as described [here](Install); follow the installation procedure until you have created a database.
+
+### Heads up to users
+Inform your users of an upcoming interruption to your service. To ensure data consistency, your server needs to be offline during some steps of the migration processes.
```

Comment:
The documentation should follow the rule "one sentence per line" which make the Markdown more easy to versionised and the look is the same when the file is rendered.

Reply:
Done.